"npm install" of stable fails, "yarn install" succeeds

$ npm install https://github.com/grommet/grommet-theme-hpe/tarball/stable
npm ERR! code ENOPACKAGEJSON
npm ERR! package.json Non-registry package missing package.json: https://github.com/grommet/grommet-theme-hpe/tarball/stable.
npm ERR! package.json npm can't find a package.json file in your current directory.

Meanwhile, replacing `npm` with `yarn` succeeds.

--- a/package.json
+++ b/package.json
@@ -1,9 +1,9 @@
 {
   "name": "grommet-theme-hpe",
   "version": "5.0.3",
-  "main": "dist/index.js",
-  "module": "dist/es6/index.js",
-  "jsnext:main": "dist/es6/index.js",
+  "main": "index.js",
+  "module": "es6/index.js",
+  "jsnext:main": "es6/index.js",
   "sideEffects": false,
   "description": "Hewlett Packard Enterprise theme for grommet",
   "authors": [
@@ -56,7 +56,7 @@
   },
   "scripts": {
     "prepublishOnly": "npm run build && npm run jsonify",
-    "build": "webpack --mode production && cross-env babel ./src/js/ --out-dir ./dist --copy-files && cross-env BABEL_ENV=es6 babel ./src/js/ --out-dir ./dist/es6 --copy-files",
+    "build": "webpack --mode production && cross-env babel ./src/js/ --out-dir ./dist --copy-files && cp LICENSE package.json dist/ && cross-env BABEL_ENV=es6 babel ./src/js/ --out-dir ./dist/es6 --copy-files",
     "lint": "eslint src",
     "lint-fix": "eslint src --fix",
     "prettier": "pretty-quick --staged",

